lib/services/robot_notifications_listener.dart

```diff
@@ -60,6 +63,17 @@ class RobotNotificationsListener {
       return;
     }
 
+    if (data.containsKey('displayTime')) {
+      displayTime = data['displayTime'];
+    }
+
+    if (data.containsKey('width')) {
+      width = data['width'];
+    }
+    if (data.containsKey('height')) {
+      height = data['height'];
+    }
```

Comment:
Surround these with `tryCast` to avoid throwing errors if the data types aren't a double